### ISSUE TITLE
Need new GH workflow to build and publish new version of builder image "eclipse-theia/blueprint"

### ISSUE THREAD

**Issue by marcdumais-work** [eclipse-theia/theia-blueprint#125]
### Bug Description:

We use [this](https://hub.docker.com/r/eclipsetheia/theia-blueprint/tags?page=1&ordering=last_updated) docker image in our [Jenkins CI](https://github.com/eclipse-theia/theia-blueprint/blob/master/Jenkinsfile#L27) (eclipsetheia/theia-blueprint:latest). There was recently a need to [update that builder image](https://github.com/eclipse-theia/theia-blueprint/pull/114), to add dependency `libsecret-1-dev`, following upstream framework changes. 

I had hoped that the image would be auto-magically updated, but it has not happened; I think we probably need to rebuild it and then publish it to `dockerhub`. IIRC the "eclipse-theia" org is an official one, controlled by the EF, so we probably need webmaster's help (e.g. to be granted write-access to be able to push the image).

Note: master is not yet affected because it still uses an older Theia verson, v1.13.0. We will not be able to update it to latest without pushing that new image. 

### Steps to Reproduce:

1. Look at this [PR](https://github.com/eclipse-theia/theia-blueprint/pull/93) that uses a recent `next` Theia
2. In the CI log, search for "libsecret-1-dev": you'll find errors about it being missing.
3. That package is included in the root [Dockerfile](https://github.com/eclipse-theia/theia-blueprint/blob/master/Dockerfile#L2)
4. run ```docker run --init -it --rm --entrypoint /bin/bash  eclipsetheia/theia-blueprint:latest```, then, in the container: ```dpkg -l | grep libsecret``` - confirm that this package is not installed in the latest published builder image

### Additional Information

- Operating System:
- Theia Version:

**Comment by marcdumais-work** [comment]
@thegecko can you refresh my memory about how you went about to be able to publish the existing version of the builder image?

**Comment by marcdumais-work** [comment]
I found this bugzilla, that's related: https://bugs.eclipse.org/bugs/show_bug.cgi?id=571598 , and within this link to EF doc: https://wiki.eclipse.org/CBI#Docker_Hub

TL;DR: it seems that committers can provide webmaster with dockerhub ID to be added to the org and be able to push. I have registered a bugzilla: https://bugs.eclipse.org/bugs/show_bug.cgi?id=574953

**Comment by thegecko** [comment]
@marcdumais-work , the image needs to be published here:

https://hub.docker.com/repository/docker/eclipsetheia/theia-blueprint

I can't see who has access to do that (beyond myself), so I believe your bugzilla ticket is the way to go.

**Comment by marcdumais-work** [comment]
Hi @thegecko,

My bugzilla has not received an answer yet, but there's apparently a problem adding new members to the (free) docker hub organisation:
![image](https://user-images.githubusercontent.com/25749063/126692929-5b3d5ccf-495d-43f2-87f5-26b1f6961f38.png)

Would you be able to build and push the new version of the Dockerfile, this time?

**Comment by marcdumais-work** [comment]
@thegecko: nevermind, I was added to the docker hub org. I guess I got lucky that there was an available spot :)

**Comment by marcdumais-work** [comment]
Correction: Me being added to the org was an intermediary step. Webmaster went ahead with a more solid solution, replacing my user with a bot and configuring the bot's credentials on our GH repo so it's available to GH actions. I'll add a workflow that we can start manually to rebuild our `Dockerfile` and push it to Docker Hub.

**Comment by marcdumais-work** [comment]
In the new workflow, the docker hub credentials should be accessible like so, and can then be used to logon:

          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}

**Comment by thegecko** [comment]
I have just pushed the latest image.
